src/Button/Button.stories.mdx

```diff
@@ -447,7 +447,7 @@ In space constrained places or for floating action buttons (FAB), we can show an
 
 ### End Icons
 
-You can specify that a button has an additional icon at the end of the row
+You can specify that a button has an additional icon at the end of the row. This will be assumed to be a menu selector and will change the button's right padding.
```

Comment:
This seems a bit unexpected? I would expect icon and endIcon to behave similarly. If `endIcon` is for a specific use case maybe the name should reflect that. 
Or remove `endIcon` and add a prop like isMenuSelect that changes the padding and moves the icon to the right?

Reply:
In retrospect I wish I'd named it something more like that; but this is already in place and now will just behave correctly 🤷‍♂️